app/boards/shields/leeloo/boards/nice_nano.overlay

```diff
@@ -0,0 +1,29 @@
+&spi1 {
```

Comment:
I don't think this file is needed, it doesn't seem that the Leeloo has RGB underglow (which is what this file is for)